query-connector/README.md

```diff
@@ -77,7 +77,36 @@ The Query Connector will eventually require other inputs from other DIBBs servic
 
 ### Developer Documentation
 
-Can be found in [api-documentation.md](api-documentation.md).
+A Postman collection demonstrating use of the API can be found [here](https://github.com/CDCgov/dibbs-query-connector/blob/main/query-connector/src/app/assets/DIBBs_Query_Connector_API.postman_collection.json).
+
+### Query Connector Data for Query Building
+
+When initializing, the Query Connector makes the value sets associated with 200+ reportable conditions available to users tasked with building queries for their jurisdiction. To group value sets by condition and to group the conditions by type, the Query Connector obtains and organizes data from the eRSD and the VSAC in the following way:
```

Comment:
Minor nit, but we might want to clarify what the "When initializing" is referring to here. E.g., is it the Database, and just the first time? The app as a whole? Maybe just some additional verbiage indicating this being a one-time process in production and mentioning that it's the app's back end postgres instance.